fastchat/serve/model_worker.py

```diff
@@ -82,6 +82,7 @@ def __init__(
             gptq_config=gptq_config,
             awq_config=awq_config,
             exllama_config=exllama_config,
+            debug=kwargs["debug"],
```

Comment:
put debug in into the function signature

Reply:
Okay